src/components/common/navBar/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './navStyles.css';
+function NavBar(props) {
```

Comment:
This is a personal practice that I do, I liked to destructuring the props like instead of use props.logo we can declare it at the functions like. function NavBar({ logo }).  and we save the word props. We have to decide how to do it to the rest of the apps.

Reply:
I am happy to do it either way, but for consistency we'll deconstruct them. It does keep everything nice and clean.